src/client/Fh.cc

```diff
@@ -18,9 +18,9 @@
 
 #include "Fh.h"
 
-Fh::Fh(Inode *in) : inode(in), _ref(1), pos(0), mds(0), mode(0), flags(0),
-                pos_locked(false), actor_perms(), readahead(),
-                fcntl_locks(NULL), flock_locks(NULL)
+Fh::Fh(InodeRef in, int flags, int cmode, const UserPerm &perms) :
+    inode(in), _ref(1), pos(0), mds(0), mode(cmode), flags(flags), pos_locked(false),
+    actor_perms(perms), readahead(), fcntl_locks(NULL), flock_locks(NULL)
```

Comment:
missed `pos_waiters` or intentionally left ?

Reply:
default ctor of a list is an empty list

Reply:
@batrick oh sorry, I misread that variable as a pointer to a list :-)